src/components/Calculator.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
- [ ] As pointed out by the previous reviewer, kindly note that you are not supposed to disable Linters. Linters are great tools that flag programming bugs. Instead of changing their rules, fix them.
- **Hint:** You may probably be struggling with Linters since you are extending `React.component.`. I'll kindly advise you to try out `React.PureComponent`, that's

 ```.js 
      class Calculator extends React.PureComponent {
 ```
[ReadMore...](https://www.geeksforgeeks.org/reactjs-pure-components/)

Reply:
Thanks a lot for your valuable help.